From dab02822f261c662edeaf9876d9fdac61214fb10 Mon Sep 17 00:00:00 2001
From: Mathias Meyer <meyer@paperplanes.de>
Date: Thu, 4 Jun 2009 16:56:12 +0200
Subject: [PATCH] Made the config_info function public.

---
 integration_api_lib.php |   14 +++++++-------
 1 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/integration_api_lib.php b/integration_api_lib.php
index 7179bea..06e6e18 100644
--- a/integration_api_lib.php
+++ b/integration_api_lib.php
@@ -62,6 +62,13 @@ class BBIntegrationApi
         return $this->config_info()->{'logout_url'};
     }
 
+    public function config_info() {
+        if (!self::$cached_config_info) {
+            self::$cached_config_info = $this->api_request("config_info");
+        }
+        return self::$cached_config_info;
+    }
+
     //------------- Private methods -------------
     protected function rails_cookie_value() {
         return $_COOKIE[$this->rails_cookie_name()];
@@ -71,13 +78,6 @@ class BBIntegrationApi
         return $this->config_info()->{'cookie_name'};
     }
 
-    protected function config_info() {
-        if (!self::$cached_config_info) {
-            self::$cached_config_info = $this->api_request("config_info");
-        }
-        return self::$cached_config_info;
-    }
-
     /**
      * Sends the API request, using HTTP_Request2. In case of an error, we issue a
      * warning, which should be trapped in an error log.
-- 
1.5.6.1.1071.g76fb

